config/management/genesis/src/storage_helper.rs

```diff
@@ -126,17 +100,21 @@ impl StorageHelper {
         command.genesis()
     }
 
-    pub fn insert_waypoint(&self, validator_ns: &str, shared_ns: &str) -> Result<Waypoint, Error> {
+    pub fn libra_root_key(
+        &self,
+        validator_ns: &str,
+        shared_ns: &str,
+    ) -> Result<Ed25519PublicKey, Error> {
         let args = format!(
             "
                 libra-genesis-tool
-                insert-waypoint
+                libra-root-key
                 --validator-backend backend={backend};\
                     path={path};\
                     namespace={validator_ns}
                 --shared-backend backend={backend};\
                     path={path};\
-                    namespace={shared_ns}
+                    namespace={shared_ns}\
```

Comment:
Any particular reason for the `\`

Reply:
that's weird... I know for a fact I did not insert that, might be some rebase issue.